libs/logging-bundle/README.md

```diff
@@ -0,0 +1,17 @@
+# Keboola Logging Bundle
+Features:
+* adds DataDog APM metadata to logs
+
+## Installation
+```shell
+composer require keboola/logging-bundle
+```
+
+## Configuration
+No configuration is required.
+
+Based on if the DataDog extension is available, the bundle will automatically register the log processor.
```

Comment:
If I just looked at this readme I wouldn't know how to `## Usage` this bundle (of course I would suppose that I'd just register it in my services.yml, but maybe should we say so?

Reply:
No, you do not need to do anything at all, that's the beauty of the bundle :slightly_smiling_face: (in the current setup, maybe there will be some configuration in the future)

Reply:
I'm scared of magic

Reply:
Not a magic, just something we were not used to yet :wink:

Reply:
aha, so just composer install and voila, cool